app/routes/surveys/components/AdminSideBar.js

```diff
@@ -15,16 +15,19 @@ type Props = {
   actionGrant: ActionGrant,
   token?: string,
   shareSurvey: number => Promise<*>,
-  hideSurvey: number => Promise<*>
+  hideSurvey: number => Promise<*>,
+  exportSurvey?: number => Promise<*>
 };
 
 type State = {
-  copied: boolean
+  copied: boolean,
+  exported: ?{ url: string, filename: string }
```

Comment:
Maybe a shitty name for an object that can be `undef` 🥇🥇??  Sound like a bool 🙃. Maybe call it something like **generatedCVC**? Makes more sense when it's used like `generatedCVC.filename` and `generatedCVC.url`.